### DISCUSSION TITLE
Easier Listen/Mute method - Ignore TG's - Dark Mode/App

### DISCUSSION THREAD

**Discussion by Grey9498** [DSheirer/sdrtrunk#1510] [Ideas]
I know this has been in the pipeline for a long time, but as we move from more powerful desktops and laptops to smaller devices, I think ignoring TG's should be a priority. It's not enough just to mute them. Because it still eats bandwidth and cpu resources and generates a lot of heat because even though you aren't LISTENING to muted groups you are still receiving/decoding/displaying them.

On a system with dozens of TG's this can be quite annoying even with them muted.

**Comment by reppiz** [comment]
I agree. I added an alias for all the talkgroups in my county, yet I still see calls come in that are not aliased? How does this happen?
